Library/Formula/leo.rb

```diff
@@ -0,0 +1,26 @@
+class Leo < Formula
+  homepage "http://leoeditor.com/"
+  url "https://downloads.sourceforge.net/projects/leo/files/Leo/5.1-final/Leo-5.1-final.zip"
+  sha256 "2d742f9825959ba5c7624d1179b9f3065e14e055c90272fbce199f91770de826"
+
+  depends_on "pyqt"
+  depends_on "enchant" => :recommended
+
+  def install
+    # Obtain information on Python installation
+    python_xy = "python" + `%x(python -c "import sys;print(sys.version[:3])")`.chomp
```

Comment:
On Thu, Apr 23, 2015 at 12:59 PM,
​​
Dominyk Tiller <notifications@github.com> wrote:

In Library/Formula/leo.rb
> <https://github.com/Homebrew/homebrew/pull/38980#discussion_r28986872>:
> ​ [snip]​
>
> It'll almost always be a Python2.x on OS X, whether that's Homebrew's or
> the System's - So you can just do (lib+"python2.7/site-packages").install
> "leo" or something similar. You can ensure it'll always be Python2.7 by
> doing depends_on :python if MacOS.version <= :snow_leopard.


​Many thanks, ​Dominyk and Ludwig, for your helpful comments​.  They are
greatly appreciated.

Edward